dataprep/eda/basic/compute.py

```diff
@@ -216,26 +223,40 @@ def compute_univariate(
                 df = df[df[x].between(value_range[0], value_range[1])]
             else:
                 print("Invalid range of values for this column", file=stderr)
-        # qq plot
+        # qq plot #qqdata[2], [3] = mean, std
         qqdata = calc_qqnorm(df[x].dropna())
         # histogram
         histdata = dask.compute(dask.delayed(calc_hist)(df[x], bins))
-        # kde plot
+        # kde plot #kdedata[3], [4] = min, max
         kdedata = calc_hist_kde(df[x].dropna().values, bins)
         # box plot
         boxdata = calc_box(df[[x]].dropna(), bins)
```

Comment:
Can you also look at passing the quantiles (qqdata[0]) to `calc_box()`? I think you can then add them to the call
``` python
    if len(df.columns) == 1:
        df = _calc_box_stats(df[x], x, qntls=qntls)
```
And then in `_calc_box_stats()` have something like
``` python 
    try:
        if qntls is None:
            if dlyd:
                qntls = np.round(grp_srs.quantile([0.25, 0.50, 0.75]), 3)
            else:
                qntls = np.round(grp_srs.quantile([0.25, 0.50, 0.75]).compute(), 3)
        stats["q1"], stats["q2"], stats["q3"] = qntls[0.25], qntls[0.50], qntls[0.75]
```